main/http_server/axe-os/src/app/components/settings/settings.component.ts

```diff
@@ -61,6 +61,7 @@ export class SettingsComponent {
         this.form = this.fb.group({
           flipscreen: [info.flipscreen == 1],
           invertscreen: [info.invertscreen == 1],
+          screenTimeout: [info.screenTimeout, [Validators.required]],
```

Comment:
I could be wrong, but I don't think it's needed here.  There use to be more in settings, but think a lot got moved to the edit component and this may need to be cleaned up?

Reply:
I guess you're right. There is no difference without this line (tested with "ng serve").
Should it be removed? Html is trial and error for me

Reply:
Really this whole form (fb) should probably be removed as I don't think its needed in this file anymore.  Shouldn't matter if you leave it or remove it, as the whole section should be cleaned up in a separate PR IMO